LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;Remortgage">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Remortgage</rdfs:label>
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-BusinessOrCommercial">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>business or commercial</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-DwellingConstruction">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>dwelling construction</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-HomeImprovement">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>home improvement</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-LoanRefinancing">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>loan refinancing</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-MortgageModification">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>mortgage modification</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-mtg;MortgageLoanPurpose-RealPropertyPurchase">
+		<rdf:type rdf:resource="&fibo-loan-typ-mtg;MortgageLoanPurpose"/>
+		<rdfs:label>real property purchase</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;NMLSR-ID">
+		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Law/LegalCapacity/LicenseIdentifier"/>
+		<rdfs:label>Nationwide Mortgage Licensing System and Registry Identifier</rdfs:label>
+		<skos:definition>the number permanently assigned by the Nationwide Mortgage Licensing System and Registry (NMLS) for each company, branch, and individual that maintains a single account on NMLS.</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>NMLSR ID</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:adaptedFrom>http://mortgage.nationwidelicensingsystem.org/about/Pages/NMLSID.aspx</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;RemortgageOnDifferentTerms">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;Remortgage"/>
-		<rdfs:label xml:lang="en">Remortgage On Different Terms</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;PropertyInspection">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-asmt;AssessmentActivity"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isProvidedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-pas-fpas;LegalAgent"/>
```

Comment:
does not have to be a LegalAgent e.g. for a termite inspection

Reply:
I changed it to party-in-role for now.  Likely it will be either a legal agent or licensee, but I don't know how many different kinds of inspections there might be, especially for commercial properties, so I left it at party-in-role.